ISL.Providers.Notifications.Abstractions.Tests.Unit/NotificationAbstractionTests.SendEmail.Exceptions.cs

```diff
@@ -28,13 +28,13 @@ public async Task ShouldThrowValidationExceptionWhenTypeINotificationValidationE
 
             NotificationProviderValidationException expectedNotificationValidationProviderException =
                 new NotificationProviderValidationException(
-                    message: "Notification validation errors occurred, please try again.",
+                    message: someNotificationValidationException.Message,
                     innerException: someNotificationValidationException);
```

Comment:
This is a false positive test.  It only pass because the data dictionary is null/ empty


```suggestion
            NotificationProviderValidationException expectedNotificationValidationProviderException =
                new NotificationProviderValidationException(
                    message: someNotificationValidationException.Message,
                    innerException: someNotificationValidationException,
                    data: someNotificationValidationException.Data);
```